benchmark/test_special_perf.py

```diff
@@ -140,6 +140,37 @@ def padding_kwargs(dtype, batch, size):
     bench.run()
 
 
+def test_perf_upsample_bicubic2d_aa():
+    def upsample_bicubic2d_aa_kwargs(dtype, batch, size):
+        channel = 16
+        scale_factors = (2, 2)
+        shape = (batch, channel, size, size)
+        output_size = (
+            int(shape[2] * scale_factors[0]),
+            int(shape[3] * scale_factors[1]),
+        )
+        input = torch.randn(size=shape, device="cuda", dtype=dtype)
+        return {
+            "input": input,
+            "output_size": output_size,
+            "align_corners": False,
+            "scales_h": None,
+            "scales_w": None,
+        }
+
+    bench = Benchmark(
+        op_name="_upsample_bicubic2d_aa",
+        # py api torch.nn.functional.interpolate,
```

Comment:
Why can't we just use the pytorch api `torch.nn.functional.interpolate` with proper `args` to make the api call the `torch._C._nn._upsample_bicubic2d_aa` functions

Reply:
pytorch api `torch.nn.functional.interpolate` is a superset of `torch._C._nn._upsample_bicubic2d_aa`, it's better to keep the same api with the atenlib function